mula/scheduler/schedulers/schedulers/normalizer.py

```diff
@@ -221,7 +216,9 @@ def has_normalizer_permission_to_run(self, normalizer: models.Plugin) -> bool:
 
         return True
 
-    def has_normalizer_task_started_running(self, task: models.NormalizerTask) -> bool:
+    def has_normalizer_task_started_running(
+        self, task: models.NormalizerTask, task_db: models.Task | None = None
```

Comment:
The `task_db` is unused and queried below again; another potential win here.

Reply:
Yes, you're right should have removed this one. Thanks!